pkg/metadata/index.go

```diff
@@ -35,7 +36,16 @@ func (t *index) List(offset, limit int) ([]*meta.Index, error) {
 		idx := new(meta.Index)
 		err = json.Unmarshal(d, idx)
 		if err != nil {
-			return nil, err
+			if err.Error() == "expected { character for map value" {
```

Comment:
I think we should rather do if `strings.Contains` here.

Reply:
Yeah, do this for the exactly error. may be you got some error else?